packages/rest-client-jsonapi/src/setupTests.ts

```diff
@@ -1,6 +1,25 @@
 import { beforeAll, afterAll, afterEach } from "vitest"
 import { server } from "./mocks/server"
 
-beforeAll(() => server.listen({ onUnhandledRequest: "error" }))
+export const testBackendEndpointConfig = {
+  api: "api",
+  schema: "articles",
+}
```

Comment:
I structured this as an object, exported it, and used it in the test file so that it's obvious why we're not handling this request endpoint